internal/server/statvar/golden/search_statvar/count_person.json

```diff
@@ -3108,6 +3160,34 @@
       "name": "People employed in the private sector without a formal contract",
       "dcid": "Count_Person_Years14Onwards_PrivateSectorNoFormalContract_Employed"
     },
+    {
+      "name": "Person Aged 15 and above",
+      "dcid": "Count_Person_15OrMoreYears_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military",
+      "dcid": "Count_Person_3OrMoreYears_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military, age 65 years and above",
+      "dcid": "Count_Person_65OrMoreYears_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military, age between 15 and 24 years",
+      "dcid": "Count_Person_15To24Years_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military, age between 25 and 44 years",
+      "dcid": "Count_Person_25To44Years_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military, age between 3 and 14 years",
+      "dcid": "Count_Person_3To14Years_Civilian"
+    },
+    {
+      "name": "Person Aged 3 and above,Not Active-Duty Military, age between 45 and 64 years",
+      "dcid": "Count_Person_45To64Years_Civilian"
```

Comment:
These came from recent NTIA import and are being changed to Count_Household_ in a followup CL as the source data was not for the all person, but the primary householder.